### PR TITLE
Migrate option comments to be consistent

### PR DESCRIPTION
**Pull request by kmadsen**
## Description

Addresses two items from https://github.com/mapbox/mapbox-navigation-android/issues/2709

- [x] I have added any issue links
- [ ] I have added all related labels (`bug`, `feature`, `new API(s)`, `SEMVER`, etc.)
- [x] I have added the appropriate milestone and project boards

## Testing

Please describe the manual tests that you ran to verify your changes

- [ ] I have tested locally (including `SNAPSHOT` upstream dependencies if needed) through testapp/demo app and run all activities to avoid regressions
- [ ] I have tested via a test drive, or a simulation/mock location app
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have updated the `CHANGELOG` including this PR

### CODE REVIEW

File: libnavigation-core/src/main/java/com/mapbox/navigation/core/stops/ArrivalController.kt [hunk 1]
```diff
@@ -46,23 +46,20 @@ class AutoArrivalController : ArrivalController {
...
-
-    /**
-     * While the next stop is less than [arrivalInMeters] away,
-     * [ArrivalController.navigateNextRouteLeg] will be called.
-     */
-    val arrivalInMeters: Double?
+    val arrivalInMeters: Double?,
+    val builder: Builder
 ) {
     /**
```
Comment (Guardiola31337) [inline libnavigation-core/src/main/java/com/mapbox/navigation/core/stops/ArrivalController.kt:61]: Should we expose `toBuilder`?

File: libnavigation-core/src/main/java/com/mapbox/navigation/core/replay/route2/ReplayRouteOptions.kt [hunk 2]
```diff
@@ -5,28 +5,21 @@ package com.mapbox.navigation.core.replay.route2
...
-     */
     val maxAcceleration: Double,
-    /**
-     * How fast the driver will decelerate in mps^2
-     */
-    val minAcceleration: Double
+    val minAcceleration: Double,
+    val builder: Builder
 ) {
     /**
```
Comment (Guardiola31337) [inline libnavigation-core/src/main/java/com/mapbox/navigation/core/replay/route2/ReplayRouteOptions.kt:24]: Should we expose `toBuilder`?

File: libnavigation-core/src/main/java/com/mapbox/navigation/core/stops/ArrivalController.kt (hunk 1 above)
Comment (kmadsen) [reply libnavigation-core/src/main/java/com/mapbox/navigation/core/stops/ArrivalController.kt:61]: yeah it is nicer for java callers

File: libnavigation-core/src/main/java/com/mapbox/navigation/core/replay/route2/ReplayRouteOptions.kt [hunk 3]
```diff
@@ -5,34 +5,32 @@ package com.mapbox.navigation.core.replay.route2
...
-    val minAcceleration: Double
-) {
+    fun toBuilder() = builder
+
     /**
      * Used to build [ReplayRouteOptions].
      */
     class Builder {
-        private var maxSpeedMps = 30.0
+        private var maxSpeedMps = 20.0
```
Comment (kmadsen) [inline libnavigation-core/src/main/java/com/mapbox/navigation/core/replay/route2/ReplayRouteOptions.kt:33]: 🔊 📢 

Slipping a default change in this pull request. There were multiple mentions that the simulated driver is too fast. Also, we have the ability to speed up the replay speed using MapboxReplayer.playbackSpeed

Comment (kmadsen) [reply libnavigation-core/src/main/java/com/mapbox/navigation/core/replay/route2/ReplayRouteOptions.kt]: failed tests 🤔  going to look into it separately

**Pull request merged**